src/test/java/pages/TestPage.java

```diff
@@ -0,0 +1,109 @@
+package pages;
+
+import io.appium.java_client.AppiumDriver;
+import io.appium.java_client.MobileElement;
+import io.appium.java_client.TouchAction;
+import io.appium.java_client.pagefactory.iOSXCUITFindBy;
+import static io.appium.java_client.touch.offset.PointOption.point;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class TestPage extends BasePage{
+
+    @iOSXCUITFindBy(xpath = "//XCUIElementTypeApplication[@name='TestApp']/XCUIElementTypeWindow[1]/XCUIElementTypeOther")
+    public MobileElement pageView;
```

Comment:
code formatting: its better to leave 1 empty line after variables for better readability